jupyterhub/jupyterhub/base/jupyterhub-dc.yaml

```diff
@@ -182,12 +182,10 @@ spec:
           requests:
             cpu: 50m
             memory: 50Mi
-        image: 'registry.redhat.io/openshift4/ose-leader-elector-rhel8:v4.7'
+        image: 'quay.io/opendatahub/leader-elector:v0.1'
```

Comment:
@tmckayus Are we going to need a new instance of the magic sed to use the official image?

Reply:
yes, testing it now